_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Snap Minis: How we can help you bring your app into SnapChat"
+date: 2020-06-20 11:00:00
+reviewed: 2020-06-20 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last week, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
```

Comment:
@etagwerker the Snap Kit API has been around, but the Snap Minis were announced as a "brand new product" at their partner summit, so I am just building on the language Snap and other publications are using.

I don't think a new set of Minis would be accurate because these didn't exist previously. The games were already there but Snap Minis are a brand new thing.

Reply:
Keep in mind we don't use Snap Kit, that's to connect one application so snapchat's services to have login/bitmojis features but on standalone apps. For the minis, the SDK is not SnapKit, it's a different JavaScript SDK called SnapCanvas and it works as a bridge between the mini and the native app (and there's no public docs, the sdk and the docs are downloaded from puppy CMS)